### PR TITLE
Add BindableSingle

### PR DESCRIPTION
**Pull request by paparony03**

### CODE REVIEW

**Comment by Tom94** [comment]
That looks good, thanks! Could you also add support for BindableSingle in the `SliderBar<T>` constructor and adjust the usage of `AddSliderStep` to use `float` instead of `double` in `TestCaseSliderBar`?

**Comment by default0** [comment]
Why is this called BindableSingle when the others are called BindableLong/BindableInt? Should be BindableFloat since were going analogous to the built-ins.

**Comment by Tom94** [comment]
Good point. Gonna fix.

File: osu.Framework/Configuration/ConfigManager.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu-framework/master/LICENCE
 
+using osu.Framework.Logging;
```
Comment (MrTheMake) [inline osu.Framework/Configuration/ConfigManager.cs]: Is there a general standard we have for using orders? I always thought that we use:
- System
- Other libraries
- osu.Framework
- osu.Game

(All of them ordered by alphabetic order)

Comment (Tom94) [inline osu.Framework/Configuration/ConfigManager.cs]: I don't think we currently follow any standard. I usually just let visual studio manage the required usings for the things I use, which in turn seems to order them somewhat arbitrarily.

**Pull request merged**